tests/no-closest.js

```diff
@@ -3,7 +3,7 @@
 const rule = require('../rules/no-closest')
 const RuleTester = require('eslint').RuleTester
 
-const error = 'Prefer closest to closest'
+const error = 'Prefer Element.closest to $.closest'
```

Comment:
!

Reply:
Indeed, fixed in situ as I was rewriting those rules.